benchmarking/config.yaml

```diff
@@ -0,0 +1 @@
+api: sambastudio
```

Comment:
Good question here: the original `README` told users to edit the YAML so that this key-value pair was set correctly, but then the repo only offered a blank `config.yaml` for them to edit. I just added the line as a convenience, since they were being asked to do it themselves anyway.
I'm happy with either potential solution here, either: 
- Continue to have the app unconditionally use Studio (at which point I'm happy to shorten the setup instructions and to delete the YAML entirely), or
- Tweak the app to actually make use of the config (I'm less comfortable with how to do this at the moment, but I'm sure I can figure it out)

Let me know which option we prefer!

Reply:
I think that eventually we'll end up having a `config.yaml` since `api` could be sambastudio, sambaverse, or other outside LLMs to benchmark e.g. OpenAI, etc. That was the vision of the kit as far as I know, but we're starting with `sambastudio`. About `mode`, we'll incorporate batch inference soon, so that'd be another option. I can take care of it since I'm more familiar with the code if you don't mind :)

Reply:
By all means, it's yours! 😄 
Shall I just revert to the original, blank config file, then?

Reply:
It's fine I'll add what is left based on this PR, thanks!